packages/block-editor/src/store/actions.js

```diff
@@ -430,8 +430,20 @@ export function replaceBlock( clientId, block ) {
  * @return {Function} Action creator.
  */
 function createOnMove( type ) {
-	return ( clientIds, rootClientId ) => {
-		return {
+	return function* ( clientIds, rootClientId ) {
+		const canMoveBlocks = yield controls.select(
```

Comment:
I wasn't sure how would mixed blocks behave, so I follow what was done with duplication. If you have several blocks selected and one of them is locked (can't be removed or moved) then the whole action is prevented.

Reply:
This seems to be the right approach 👍